### PR TITLE
feat: server implementation

### PR DESCRIPTION
**Pull request by hannahmkim**

### CODE REVIEW

File: server/src/server.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import express from 'express';
```
Comment (d48) [inline server/src/server.js:1]: i get an error when trying to run this locally for some reason. I `yarn install` and `yarn start` and get this error below. I switched to my mac btw.

```bash
╰─$ yarn start
yarn run v1.19.1
$ node src/server.js
/Users/ryanregalado/Dev/repos/lucent/server/src/server.js:1
import express from 'express';
       ^^^^^^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Comment (hannahmkim) [reply server/src/server.js:1]: i believe es6 imports only work with node version > 13. whats your node version?

Comment (d48) [reply server/src/server.js:1]: ah, my node version is `10.16.3`. I'll update right now and test.

Comment (d48) [reply server/src/server.js:1]: awesome, just tested in `v13.12.0` and it worked. great work

**Pull request merged**